### PR TITLE
Added tests for movement module and a bug

### PR DESCRIPTION
**Pull request by ctlai95**

### CODE REVIEW

**Review by henrytran28 — CHANGES_REQUESTED** [review]

**Comment by ctlai95** [comment]
For the I-spins I followed this instead: http://tetris.wikia.com/wiki/I-spins_in_SRS
which all passed so i think we're good :)

**Comment by ctlai95** [comment]
tests failed gg

File: src/movement/movement_test.py [hunk 1]
```diff
@@ -0,0 +1,863 @@
...
+def test_rotation_twists():
+    for twist_test in twist_test_list:
+        movement.board.current_tetromino = Tetromino(
+            twist_test.tetromino_id, twist_test.initial_position, colors.ASH)
+        if twist_test.initial_rotation is not None:
+            twist_test.initial_rotation()
+        if twist_test.initial_offset is not None:
+            twist_test.initial_offset()
+        assert (movement.board.current_tetromino.origin.x,
+                movement.board.current_tetromino.origin.y) == (twist_test.initial_position.x, twist_test.initial_position.y)
```
Comment (henrytran28) [inline src/movement/movement_test.py]: Use the equals function

File: src/movement/movement_test.py [hunk 2]
```diff
@@ -0,0 +1,863 @@
...
+        assert movement.board.current_tetromino.state == twist_test.initial_state
+        for board_square_position in twist_test.boundary_positions:
```
Comment (henrytran28) [inline src/movement/movement_test.py]: boundary position

File: src/movement/movement_test.py [hunk 3]
```diff
@@ -0,0 +1,863 @@
...
+
+
+def test_hard_drop():
+    for tetromino_id in tetrominos_id_list:
+        for i in range(movement.board.width - get_tetromino_width(tetromino_id)):
+            for j in range(movement.board.height - get_tetromino_height(tetromino_id)):
+                movement.board.current_tetromino = Tetromino(
+                    tetromino_id, Point(i, j), colors.ASH)
+                position_before_move = movement.board.current_tetromino.origin
+                print(movement.board.get_combined_matrix_string())
```
Comment (henrytran28) [inline src/movement/movement_test.py]: delet this nephew

**Review by henrytran28 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**